appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.service.js

```diff
@@ -16,3 +16,33 @@ export const getAttachmentsFolder = async (apiClient, appealId) => {
 
 	return folders[0];
 };
+
+/**
+ * @param {{ firstName: string, lastName: string, emailAddress: string, addressLine1: string, addressLine2: string, town: string, county: string, postCode: string, redactionStatus: boolean, 'date-day': string, 'date-month': string, 'date-year': string }} comment
+ * @param {import('got').Got} apiClient
+ * @param {string} document
+ * @param {number} appealId
+ * @returns {Promise<import('@pins/appeals.api').Appeals.FolderInfo>}
+ * */
+export const createIPComment = async (comment, document, apiClient, appealId) => {
+	const json = {
+		ipDetails: {
+			firstName: comment.firstName,
+			lastName: comment.lastName,
+			email: comment.emailAddress
+		},
+		ipAddress: {
+			addressLine1: comment.addressLine1,
+			addressLine2: comment.addressLine2,
+			town: comment.town,
+			county: comment.county,
+			postCode: comment.postCode
+		},
+		attachments: [document],
+		redactionStatus: comment.redactionStatus
+	};
+
+	const IPcomment = await apiClient.post(`appeals/${appealId}/reps/comments`, { json }).json();
```

Comment:
Very minor point, but this should probably be camel case i.e. `ipComment` - even though capitalised `IP` is grammatically correct here, my personal view is consistency should probably trump that (and capitalised variables are syntax highlighted differently, which while not an issue itself, could potentially cause confusion)